etc/pip-requires-test

```diff
@@ -6,7 +6,7 @@ pytest-xdist~=2.2.0; python_version >= '3.6'      # Used for parallel testing on
 pyflakes==2.1.1                                   # Passive checker of Python programs
 flake8==3.7.8                                     # Wrapper around PyFlakes&pep8
 pylint==1.9.4; python_version < '3.6'             # static code analysis. 1.9.5 last 2.7 compatible release
-pylint==2.6.0; python_version >= '3.6'
+pylint==2.8.2; python_version >= '3.6'
```

Comment:
Ah apologies, I remember I promised a PR for this too and forgot.. Thanks for adding this :-)